tests/common/fixtures/conn_graph_facts.py

```diff
@@ -16,6 +16,8 @@ def fanout_graph_facts(localhost, duthosts, rand_one_dut_hostname, conn_graph_fa
     duthost = duthosts[rand_one_dut_hostname]
     facts = dict()
     dev_conn = conn_graph_facts.get('device_conn', {})
+    if not dev_conn:
+        return None
```

Comment:
Why return None? Some other places will use dev_conn which is expected to be a dict, no matter it is empty.

Reply:
Yes, that would be better, fixed

Reply:
I mean, `if not dev_conn:` is unnecessary. dev_conn will return `{}` if there is no key `device_conn` in conn_graph_facts

Reply:
no, this one is get fanout facts, it will directly use dev_conn[duthost.hostname].items() in next line and return keyerror vlab-01 in KVM test